spog/api/src/config/default.yaml

```diff
@@ -68,3 +70,67 @@ bombastic:
               - 'supplier:"Organization: Red Hat"'
 
 vexination:
+
+  filters:
+    categories:
+      - label: Severity
+        options:
+          - id: low
```

Comment:
This looks to me like something that would be really hard to configure. Maybe we can just use a list here and generate label and terms with a macro (or something similar)?

Reply:
That should be configurable during runtime, what's in there is just the default if nothing was configured.

Reply:
Yeah, I don't want to block PR due to this ... But just, any complex configuration with html snippets like that seems to me very error-prone ... and I'm not sure who would change the layout of the page using the config